tests/ci/ci_config.py

```diff
@@ -779,16 +794,20 @@ def validate(self) -> None:
         ),
     },
     other_jobs_configs={
+        JobNames.MARK_RELEASE_READY: TestConfig(
+            "", job_config=JobConfig(mandatory_for_release=True)
+        ),
         JobNames.DOCKER_SERVER: TestConfig(
             "",
             job_config=JobConfig(
+                mandatory_for_release=True,
```

Comment:
I am not sure if it's mandatory. Rather no. It can be done after the release too.

Besides, it won't be included in DOCKER_KEEPER, right?

Reply:
`mandatory_for_release` means that job must run on master and release branch and cannot be skipped there if never done on this branch before. I'll rename this property to `required_on_release_branch` i hope it will be more clear then